### ISSUE TITLE
JAWS doesn't announce link text or role when the link is nested inside a checkbox or radio button's label

### ISSUE THREAD

**Issue by dabramsOS** [FreedomScientific/VFO-standards-support#960]
When navigating through a form in Google Chrome, if a text link is nested inside the `<label>` element for a checkbox or radio button, the link's semantics seem to be completely ignored by JAWS.
* Using down arrow to navigate through the page, the label text is skipped completely.
* Using tab to move between interactive elements, the link is focused, but the role and name of the previously focused form element is announced instead (if that form element was a checkbox or radio button).

Reproducible on this [CodePen](https://codepen.io/editor/dabrams888/pen/019fc95f-023a-7a93-9bea-789bcb724304) (also pasted below just in case).

In Firefox, this issue does not occur. Though the link's information seems to be announced twice for some reason when focused.

```html
<h1>Testing links nested inside field labels</h1>
<button>Tab stop</button>

<h2>Text input</h2>
<label for="input-1">Favorite food <a href="#">test</a>:</label>
<input type="text" id="input-1">

<h2>Checkbox</h2>
<input type="checkbox" id="input-2">
<label for="input-2">I consent <a href="#">test</a>.</label>

<h2>Radio button</h2>
<input type="radio" name="onoff" id="input-3">
<label for="input-3">On <a href="#">test</a></label>
<input type="radio" name="onoff" id="input-4">
<label for="input-4">Off <a href="#">test</a></label>

<h2>Textarea</h2>
<label for="input-5">Tell me a story <a href="#">Test</a></label>
<textarea id="input-5"></textarea>
```
